internal/services/formatters/generic/trivy/formatter_test.go

```diff
@@ -29,6 +31,15 @@ import (
 )
 
 func TestTrivyParseOutput(t *testing.T) {
+	dirName := filepath.Join(".horusec", "00000000-0000-0000-0000-000000000000")
+	err := os.MkdirAll(dirName, 0o777)
+	assert.NoError(t, err)
+	_, err = os.Create(filepath.Join(dirName, "go.sum"))
+	assert.NoError(t, err)
+	t.Cleanup(func() {
+		err = os.RemoveAll(".horusec")
+		assert.NoError(t, err)
+	})
```

Comment:
Why this is needed?

Reply:
now we return and fail the analysis if we don't find the file, so i had to add some mock file to test continues as the way it is.